yt/visualization/plot_modifications.py

```diff
@@ -463,6 +466,14 @@ def __init__(
 
         self.plot_args = plot_args
 
+        if None not in (lower_threshold, upper_threshold):
+            threshold_field = self.field
+        else:
+            threshold_field = None  # Need to specify at least one value
+        self.threshold_field = threshold_field
```

Comment:
i think I need more coffee this morning... I'm really struggling with mentally parsing the `if None not in (lower_threshold, upper_threshold)` line. But is the aim of this block is to set the `threshold_field` to `None` if both  `lower_threshold` and `upper_threshold` are `None`?

Reply:
This line is wrong!  Thank you!